### DISCUSSION TITLE
Current Transformer Wiring

### DISCUSSION THREAD

**Discussion by alschroder** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#149] [Q&A] [answered]
I have a question about how CTs are wired (I couldn't find anything when searching).  CTs have two output wires, or at least the one in my hand does, and the interface for the CS boards are 3-pole (or 3-terminal) 3.5mm audio jacks, or at least that's how the pre-wired CTs come.  At the risk of taking one apart, which terminals are used? I.e. how is the CT wired to the 3.5mm audio-style plug?

Thanks!

**Comment by ICPete** [comment, marked as answer]
Al, I think I can answer this one!
Here is the Circuit Setup schematic for the CT jack:
![image](https://github.com/CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter/assets/136994470/90bea3be-7a37-4d47-b7f9-06d017018fed)
And here is the pinout diagram from CUI for this component:
![image](https://github.com/CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter/assets/136994470/8cbe68b6-15d5-40cc-a5d5-e433da616bbb) 
[far right diagram]

So pins 1 & 2 are the winding connections, and pin 1 is the sleeve while pin 2 is the tip.
Hence the middle "pole" (the "ring") is not connected.

**Reply by ICPete** [reply]
BTW, also note that pin 10 of the jack is connected to pin 2 when NOTHING is plugged into it.
It disconnects as soon as you plug in the plug.
[It's for when you want to, for example, disconnect a speaker when the earphone is plugged in.]
So for the CS design only pins 1 & 2 are used, and in the CT plug only the sleeve and tip are connected to the winding.

**Reply by alschroder** [reply]
Awesome, thanks Pete. Off to the test bench.
